pages/index.tsx

```diff
@@ -106,14 +114,60 @@ export default function Home() {
     balanceStatuses[balanceStatuses.length - 1]
   )
 
+  const monthStartDate = startOfMonth(currentDate)
+  const monthEndDate = endOfMonth(currentDate)
+  const currentMonthOccurences = generateTransactionConfigsOccurances(
+    transactions,
+    monthStartDate,
+    monthEndDate
+  )
+
+  const yearStartDate = startOfYear(currentDate)
+  const yearEndDate = endOfYear(currentDate)
+  const currentYearOccurences = generateTransactionConfigsOccurances(
+    transactions,
+    yearStartDate,
+    yearEndDate
+  )
+
+  const currentYearBalanceAmount = currentYearOccurences.reduce((res, cur) => {
+    return res + cur.amount
+  }, 0)
+  const currentMonthBalanceAmount = currentMonthOccurences.reduce(
+    (res, cur) => {
+      return res + cur.amount
+    },
+    0
+  )
+
   return (
     <Layout>
       <div className={styles.status}>
         <div className={styles.first}>
-          <Ticker label="Current Balance" number={currentBalanceAmount} />
-          <Link href="/balance">
-            <div className={styles.balanceDisclaimer}>Not your balance?</div>
-          </Link>
+          <Ticker label="Balance" number={currentBalanceAmount} />
+          <div className={styles.smallTickers}>
+            <Ticker
+              small
+              label="This Year"
+              prefix={currentYearBalanceAmount < 0 ? '-' : '+'}
+              number={currentYearBalanceAmount}
+              duration={4}
+            />
+            <Ticker
+              small
+              label="This Month"
+              prefix={currentMonthBalanceAmount < 0 ? '-' : '+'}
+              number={currentMonthBalanceAmount}
+              duration={4}
+            />
+            <Ticker
+              small
+              label="Month AVG"
+              prefix={currentYearBalanceAmount < 0 ? '-' : '+'}
+              number={currentYearBalanceAmount / 12}
+              duration={4}
+            />
+          </div>
```

Comment:
how it works with next feature of display the chart bu year/month?

Reply:
I don't understand what you're asking